src/filterwidget.h

```diff
@@ -75,11 +75,22 @@ class QDLLEXPORT FilterWidget : public QObject
   void setFilteringEnabled(bool b);
   bool filteringEnabled() const;
 
+  void setFilteredBorder(bool b);
+  bool filteredBorder() const;
+
   FilterWidgetProxyModel* proxyModel();
+  QAbstractItemModel* sourceModel();
 
+  [[deprecated("use mapToSource()")]]
   QModelIndex map(const QModelIndex& index);
```

Comment:
Can't we just remove it? Since I doubt there are plugins outside of the official plugins that use this currently...